README.md

```diff
@@ -17,19 +17,49 @@ conda activate c3po-os-api
 
 7. Install dependencies and dev dependencies locally from the `Pipfile` by running `pipenv install --dev`.
```

Comment:
append: (Current workaround for intel platforms: `pip install -r requirements.txt`, pip install additional packages needed for new development, notify Eric.)

Reply:
Added clarification in readme